src/emulate.c

```diff
@@ -374,6 +374,24 @@ int emulate_common(unsigned long syscall, unsigned long *parameters,
   SYSCALL2(os_endorsement_get_public_key_certificate, "(%d, %p)",
            unsigned char,   index,
            unsigned char *, buffer);
+  
```

Comment:
There are 2 trailing whitespaces on this line, please remove them and it should be good :)

Reply:
done